### PR TITLE
Make code consistent with comment 

### PR DESCRIPTION
**Pull request by aymanhab**
Utility claims visualizer will be displayed (to test installation integrity) but UseVisualizer was not set

Fixes issue #0

### Brief summary of changes
Uncommented line that made the code and user message inconsistent

### Testing I've completed
Ran utility in dev environment and a visualizer window was opened.

### Looking for feedback on...
Should we open an issue and include in change log?
### CHANGELOG.md (choose one)

- no need to update because...
- updated...

The Doxygen for this PR can be viewed at http://myosin.sourceforge.net/?C=N;O=D; click the folder whose name is this PR's number.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2725)

### CODE REVIEW

**Comment by aymanhab** [comment]
@chrisdembia this PR is not part of the test suite, however all platform tests fail because of simbody, any clues? Thanks

**Comment by aymanhab** [comment]
Thanks @carmichaelong

**Comment by carmichaelong** [comment]
Oh weird, I thought at first the failures were random since this was such a small change. Sorry, I should have been more careful there. Looks like consistent failure with `testExampleModelRuns `

**Comment by chrisdembia** [comment]
I think we need to disable that line if the test is run as part of the test suite. We use the environment variable `OPENSIM_USE_VISUALIZER` in for the Matlab tests to handle this; we could also use this for `testExampleModelRuns`.

https://github.com/opensim-org/opensim-core/blob/048d13753a6d1f153e5481984a4576bfd65be218/Bindings/Java/Matlab/tests/CMakeLists.txt#L53-L54

File: OpenSim/Examples/checkEnvironment/checkEnvironment.cpp [hunk 1]
```diff
@@ -204,7 +204,7 @@ int main()
         // PERFORM A SIMULATION //
         /////////////////////////
         
-        //osimModel.setUseVisualizer(true);
+        osimModel.setUseVisualizer(true);
```
Comment (chrisdembia) [inline OpenSim/Examples/checkEnvironment/checkEnvironment.cpp:207]: Next, use the environment variable in the code here.

**Comment by aymanhab** [comment]
@chrisdembia the error below indicates failure before launching the visualizer, is that the expected behavior? I expect seg-fault or timeout. 

simbody-visualizer: error while loading shared libraries: libSimTKsimbody.so.3.8: cannot open shared object file: No such file or directory

**Comment by chrisdembia** [comment]
This is probably not the expected behavior. I don't know the reason for this error message.

**Comment by aymanhab** [comment]
Thanks for the feedback @chrisdembia, ready for quick review. 
We need to test an artifact built off master after merge to check visualization functionality.

**Comment by chrisdembia** [comment]
My understanding is it's an example that helps users test if OpenSim is installed properly.

**Comment by nickbianco** [comment]
Ah okay, like a self-test.

**Comment by aymanhab** [comment]
Thanks @carmichaelong @nickbianco and @chrisdembia 
It is a utility that we distribute so that users can test if the installation was functional. We include it in tests out of abundance of caution to make sure the code compiles and runs but the actual visualization needs to be checked manually in an installed version.

**Pull request merged**